LINK/usr/bin/manageiq-db-ready

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env ruby
+
+require "yaml"
+require "sd_notify"
+require "manageiq-password"
+
+$stdout.sync = true
+
+def wait_for_service(host, port)
+  loop do
+    `ncat #{host} #{port} </dev/null 2>/dev/null`
+    break if $?.success?
+
+    puts "#{host} #{port} - not accepting connections"
+    sleep(10)
+  end
+
+  puts "#{host} #{port} - accepting connections"
+end
+
+def wait_for_database(db_yaml, env = "production")
+  host, port = db_yaml[env].values_at("host", "port")
+  wait_for_service(host, port || 5432)
+end
+
+def wait_for_manageiq_db(db_yaml, env = "production")
+  host, port, dbname, password = db_yaml[env].values_at("host", "port", "database", "password")
+  password = ManageIQ::Password.decrypt(password)
```

Comment:
Super minor, but I think this might be a little more defensive, just in case someone drops a raw password right in the yaml.

```suggestion
  password = ManageIQ::Password.try_decrypt(password)
```

Reply:
:+1: